docs/en/source/v2endpoint.rst

```diff
@@ -0,0 +1,139 @@
+***********************************
+Scorpio NGSI V2 to NGSI-LD endpoint
+***********************************
+
+Scorpio provides an endpoint for NGSI V2 context producers. This endpoint will attempt to transform the provided V2 entities. 
+After the transformation process scorpio will use NGSI-LD upsert to store or update the LD versions of the entities. This means this endpoint can be used to create entities as well as update or append new attributes to existing entities.
+
+Translation Logic
+#################
+Since V2 and LD don't have 100% compatible concepts certain assumptions had to be made to allow this automated transformation.
+
+ - ID:
+If the provided ID is not a URI it will be prefixed with urn:v2told: making it a URI.
```

Comment:
update v2told --> v2ld

Reply:
If you formally wish to create a proper *Uniform Resource Name*, these are defined in the form:

```text
urn:<namespace-identifier>:<namespace>:....
```

According to Annex **A.3 NGSI-LD namespace** of the [NGSI-LD specification](https://www.etsi.org/deliver/etsi_gs/CIM/001_099/009/01.07.01_60/gs_cim009v010701p.pdf)

> EXAMPLE: `urn:ngsi-ld:Person:28976543`.
> It is recommended that applications use this URN namespace when applicable. 

_"This namespace"_ being `urn:ngsi-ld:`. So my interpretation of converting v2 data using a non-URN `id` to URN has always been to use the prefix `urn:ngsi-ld:`